pkg/api/validation/validation.go

```diff
@@ -283,6 +283,20 @@ func validateGCEPersistentDisk(PD *api.GCEPersistentDisk) errs.ValidationErrorLi
 	return allErrs
 }
 
+func ValidatePersistentVolumeName(name string, prefix bool) (bool, string) {
+	return util.IsDNS1123Label(name), name
+}
+
+func ValidatePersistentVolume(persistentvolume *api.PersistentVolume) errs.ValidationErrorList {
+	allErrs := ValidateObjectMeta(&persistentvolume.ObjectMeta, false, ValidatePersistentVolumeName)
```

Comment:
There are more fields... either validate or leave a TODO (and plan to come back to it)